### DISCUSSION TITLE
关于选择训练项目的思路

### DISCUSSION THREAD

**Discussion by a2823kevin** [NateScarlet/auto-derby#14] [Q&A] [answered]
请问能否请教下作者是根据什么样的模式(Ex:目前马娘的5数值、训练项目的加成等等)来选择训练项目?

**Comment by NateScarlet** [comment, marked as answer]
给每个训练评分 当前数值越高评分系数越低 最后选评分最高的那个 如果最高分也没达到期望值就休息 期望值在满体力时减半

https://github.com/NateScarlet/auto-derby/blob/f87a2eb20570c7cb1fdf5247bb5763630f499f69/auto_derby/jobs/nurturing/nurturing.py#L49-L121

应用了我总结了的几个规则

1. 速度越高越好
2. 耐力在速度达标的情况下标准要随着时间推移慢慢变高
3. 力量在速度达标的情况下越高越好
4. 毅力在速度达标的时候越高越好 但是高于400意义不大
5. 智力在速度达标的时候越高越好 但是高于400意义不大，比毅力优先，并且能回体力有额外评分

**Reply by a2823kevin** [reply]
大概了解了，感谢答覆!
